### PR TITLE
User Story 2

### PR DESCRIPTION
**Pull request by IsabellUrban**

### CODE REVIEW

File: components/ArtPiecePreview.js [hunk 1]
```diff
@@ -38,6 +27,6 @@ export default function ArtPiecePreview({ piece }) {
       </ImageWrapper>
       <h3>Title: {name}</h3>
```
Comment (ahohnsen) [inline components/ArtPiecePreview.js:28]: Looks like, there is an `h3` following an `h1`. In general it is better practice to use headings in a hierarchical order for clarity and structure, meaning an `h1` should follow an `h2`.

File: components/ArtPieces.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ const StyledList = styled.li`
   list-style: none;
 `;
 
-export default function ArtPiece({ pieces }) {
+export default function ArtPieces({ pieces }) {
   return (
     <>
       <h1> All art pieces:</h1>
```
Comment (ahohnsen) [inline components/ArtPieces.js:11]: There should only be one `h1` element per page.

I recommend the following structure:

- `h1` for your app title
- `h2` for the different sections of your page (spotlight and art pieces list)
- `h3` for the details, e.g. title

Comment (ahohnsen) [reply components/ArtPieces.js:11]: In my opinion, the name `StyledList` is a bit misleading (line 15). Your `StyledList` is acutally the `li` element and not the `ul` element. Therefore I suggest to use `StyledListItem` instead.

File: pages/index.js [hunk 3]
```diff
@@ -9,7 +9,7 @@ export default function HomePage() {
     `https://example-apis.vercel.app/api/art`,
     fetcher
   );
-  console.log("data", data);
+  // console.log("data", data);
```
Comment (ahohnsen) [inline pages/index.js:12]: Please remove all `console.log()` 🧹

**Review by ahohnsen — APPROVED** [review]
Well done. 🚀 

I added some comments about naming and your HTML structure. Since these are minor issues and you will be splitting your content across two pages in the next user story, you could also address these things in the next user story.

**Comment by IsabellUrban** [comment]
Thanks Andrea for your comments. We'll change them in the next user Story. :-)

**Pull request merged**